airflow/www/views.py

```diff
@@ -798,11 +798,14 @@ def log(self):
         if PY2 and not isinstance(log, unicode):
             log = log.decode('utf-8')
 
+        if request.is_xhr:
```

Comment:
im not completely agree to the concern that it'll break if we move away from `$.get`.. as in case if we do that, we'll take care of backend as well.. its not like someone just gonna change to UI code without altering the backend code as well.. 

the new api and extra params are overkill for something very simple like that..